http-gateway/web/src/containers/things/things-i18n.js

```diff
@@ -155,13 +163,22 @@ export const messages = defineMessages({
   resourceWasDeleted: {
     id: 'things.resourceWasDeleted',
     defaultMessage:
-      'The resource delete was successfully scheduled. you will be notified when the resource was deleted.',
+      'The resource delete was scheduled, you will be notified when the resource was deleted.',
   },
   resourceDeleteError: {
     id: 'things.resourceDeleteError',
     defaultMessage: 'Failed to delete a resource',
     description: 'Title of the toast message on resource delete error.',
   },
+  shadowSynchronizationError: {
+    id: 'things.shadowSynchronizationError',
+    defaultMessage: 'Failed to set shadow synchronization',
+    description: 'Title of the toast message on shadow synchronization set error.',

```

Comment:
Will this be set later?

Reply:
There are two types of warning titles: `shadowSynchronizationError` and `shadowSynchronizationWasSetOffline`. The one named "offline" is the one which will be set later, the first one is shown, when something went wrong while the device was online.